composer.json

```diff
@@ -41,11 +42,12 @@
         "symfony/security-bundle": "4.4.*",
         "symfony/swiftmailer-bundle": "^3.1",
         "symfony/templating": "4.4.*",
+        "symfony/twig-pack": "^1.0",
         "symfony/validator": "4.4.*",
         "symfony/webpack-encore-bundle": "^1.1",
         "symfony/yaml": "4.4.*",
         "twig/extensions": "^1.5",
-        "willdurand/hateoas-bundle": "^1.1"
+        "willdurand/hateoas-bundle": "^2.1"
```

Comment:
Maybe make this 2.0, so we know that we specifically needed 2.1 (or higher)

Reply:
^2.1 does mean that we specifically needed 2.1 or higher right?

Reply:
Yes, it does mean that. But this makes me thing that 2.0 was no good. What is there is no 2.1? It was more of a type-o I think.